### DISCUSSION TITLE
Select Grouping search result

### DISCUSSION THREAD

**Discussion by nazarii-rabbit** [filamentphp/filament#13118] [Help]
### Package

Other

### Package Version

v3

### How can we help you?

I have select, with grouped options. For example, i have Post, with 2 categories, and categories has same tags. So, this tags can be with same name, and if i type something in search select field - i get 2 same tags from 2 groups. How i can group search results?

![image](https://github.com/filamentphp/filament/assets/170324844/242ac6d9-2c7f-4777-8cc1-99c7276acb83)

**Comment by leandrocfe** [comment]
maybe you could customize the result adding the category
```php
->getOptionLabelFromRecordUsing(fn (Model $record) => "{$record->tag_name} ({$record->category_name})")
```

**Reply by nazarii-rabbit** [reply]
Not quite that. I mean when i type something in search i also need see grouped fields...

- Posts
  --new
  --draft
  ...
- Category
  --new
  --draft
  ...

**Reply by leandrocfe** [reply]
Hum, not sure if it is possible
